lib/diagnostic.js

```diff
@@ -6,29 +6,51 @@
 // splits it into an array of normalized words
 // (uppercase strings without punctuation)
 // and returns that array.
-const normalizeWords = function (/* */) {
-  /* your response here */
+const normalizeWords = function (paragraph) {
+  let words = paragraph.split(' ');
+  let normalizedWords = words.map((word) => {
+    return word.toUpperCase().replace(/\W+/, '');
+  });
+  return normalizedWords;
 };
 
 // 2. Write a function that takes a string argument
 // and returns an array of unique normalized words.
-const uniqueWords = function (/* */) {
-  /* your response here */
+const uniqueWords = function (arrayOfWords) {
+  let uniqueWordsAsKeys = {};
+  for (let i = 0; i < arrayOfWords.length; i++) {
```

Comment:
The argument for this function should be a string, not an array of strings.  In its current state, this loop iterates through each character of the input string, rather than each element of an array.